requirements-conda.txt

```diff
@@ -0,0 +1,17 @@
+appdirs==1.4.4
+cached-property==1.5.2
+cftime==1.6.0
+gmsh-interop==2021.1.1
+h5py==3.3.0
+importlib-metadata==4.11.3
+pybind11==2.9.2
+pybind11-global==2.9.2
+PySide2==5.13.2
+pytools==2022.1.7
+PyYAML==6.0
+Shapely==1.8.2
```

Comment:
Think now that BLUEPRINT is gone, shapely may no longer be directly required. Not sure if it is needed by any other dependencies though.

Reply:
Yeah I tried that and something pulls it in (possibly meshio..)